### ISSUE TITLE
Error: No metadata files found for isopack at

### ISSUE THREAD

**Issue by IDCOLL** [pbastowski/angular-meteor-babel#21]
I am trying to execute, meteor add angular. I get the error message as stated in the subject in my console. Can you please assist.

I will drop the entire I receive below:

**C:\Users\Ixx Cxxxxx\AppData\Local.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\isopackets\ddp\npm\node_modules\meteor\promise\node_modules\meteor-promise\promise_server.js:116
      throw error;
            ^
Error: No metadata files found for isopack at: /C/Users/Ixx Cxxxxx/AppData/Local/.meteor/packages/pbastowski_angular-babel/1.0.9
    at Isopack._loadUnibuildsFromPath (C:\tools\isobuild\isopack.js:903:13)
    at C:\tools\packaging\tropohouse.js:514:19
    at Array.forEach (native)
    at Function._.each._.forEach (C:\Users\Ixx Cxxxxx\AppData\Local.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\underscore\underscore.js:79:11)
    at C:\tools\packaging\tropohouse.js:513:11
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at Object.download (C:\tools\packaging\tropohouse.js:448:20)
    at C:\tools\packaging\tropohouse.js:593:22
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at [object Object].downloadPackagesMissingFromMap (C:\tools\packaging\tropohouse.js:590:20)
    at C:\tools\project-context.js:725:25
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at C:\tools\project-context.js:724:20
    at C:\tools\packaging\catalog\catalog.js:100:5
    at C:\tools\utils\buildmessage.js:271:13
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:264:29
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:262:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:253:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at Object.catalog.runAndRetryWithRefreshIfHelpful (C:\tools\packaging\catalog\catalog.js:99:31)
    at ProjectContext._.extend._downloadMissingPackages (C:\tools\project-context.js:723:13)
    at C:\tools\project-context.js:278:9
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at ProjectContext._completeStagesThrough (C:\tools\project-context.js:268:18)
    at C:\tools\project-context.js:260:12
    at Function.run (C:\tools\tool-env\profile.js:489:12)
    at ProjectContext.prepareProjectForBuild (C:\tools\project-context.js:259:13)
    at C:\tools\cli\commands-packages.js:2021:20
    at C:\tools\utils\buildmessage.js:271:13
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:264:29
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:262:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:253:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at Command.main.registerCommand.name [as func](C:toolsclicommands-packages.js:2020:27)
    at C:\tools\cli\main.js:1408:23**

**Comment by pbastowski** [comment]
Hi @IDCOLL 
Without your repo for me to look at it is difficult for me to help you. Could you please add a link to your repo here?

**Comment by IDCOLL** [comment]
Hi @pbastowski,
Thanks for the reply. I am very new to meteor and angular and don`t really know what you mean with repo....

I have however managed to fix the issue:
1. The issue occurred when following the Angular-Meteor tutorial.
2. Removing blaze-html-templates and ecmascript and then adding angular lead to the problem.
3. What I didn`t realize is that the actual issue was me upgrading to meteor 1.3.
4. To fix the problem I completely removed a few packages from my Appdata\Local.meteor\packages directory, including the pbastowski-angular package.
5. I then ran meteor from my cmd to reload the packages.
6. Added angular to my package, but I then got another issue.
7. To fix this issue I had to remove the standard server/main.js file that was being automatically created when using meteor create "app".

Hope it makes sense.
